test/integration/project-page.test.js

```diff
@@ -177,3 +180,69 @@ describe('www-integration project-page signed in', () => {
         await expect(gfVisible).toBe(true);
     });
 });
+
+describe('www-integration project-creation signed in', () => {
+    beforeAll(async () => {
+        // expect(projectUrl).toBe(defined);
+        driver = await buildDriver('www-integration project-creation signed in');
+        await driver.get(rootUrl);
+        await driver.sleep(1000);
+        await signIn(username, password);
+        await findByXpath('//span[contains(@class, "profile-name")]');
```

Comment:
what does this line do?

Reply:
This section 

- 188 loads the home page
- 189 waits 1 second (there are some race condition here and this is the easiest way around it)
- 190 signs in.  This is a function defined in selenium-helpers.  https://github.com/LLK/scratch-www/blob/develop/test/integration/selenium-helpers.js#L154 It clicks sign in in the header, types the username, then types the password.  As part of typing the password the last character is the return character.  It then waits until the profile name appears on the page.
- 191 this waits until the profile name appears on the page (in the upper right).  We likely don't need to do this because it has just done that, but I remember having trouble with timing in this area, so it might be helping some how.